### DISCUSSION TITLE
[Feature] add a button to show/hide local photos in the app

### DISCUSSION THREAD

**Discussion by indam** [immich-app/immich#5092] [Feature Request]
### The feature

Please add a switch that can quickly hide the local photos which were not uploaded to the cloud.

For me, the photos inside my phone contains two parts, one is my family, and the other is for working purpose. But I use Immich only for family. So I back up the photos to Immich mannually and selectively. When I browse through the timeline and want to have some leisure time, the working photos, which are not uploaded, always bursted into my eyes. That really annoys me.

So please add a way to quickly filter those annoying photos out, thank you very much. 

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by jdumeez** [comment]
I have made a similar request under #4990
